eng/Version.Details.xml

```diff
@@ -93,6 +89,14 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>3c523a6a7a3ebc25fe524359127b1d8846e23ea3</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.WindowsDesktop.App" Version="5.0.0-preview.1.20127.5">
```

Comment:
Is this up to date? Worth doing an update now if not? So we're not waiting on maestro as much?

Reply:
Yeah, it's up-to-date. 
```
λ darc get-asset --name Microsoft.WindowsDesktop.App --channel ".NET 5 Preview 1" --max-age 5
Looking up assets with name 'Microsoft.WindowsDesktop.App' on channel '.NET 5 Preview 1' in the last 5 days
Microsoft.WindowsDesktop.App @ 5.0.0-preview.1.20127.5
Repository:    https://github.com/dotnet/windowsdesktop
Branch:        release/5.0-preview1
Commit:        3220fe28e26c10885932b29f653b573f7d845b31
Build Number:  20200227.5
Date Produced: 2/27/2020 5:27 PM
Build Link:    https://dev.azure.com/dnceng/internal/_build/results?buildId=539885
BAR Build Id:  44403
Released:      False
Channels:
- .NET 5 Preview 1
Locations:
- https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json (NugetFeed)
- https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5-transport/nuget/v3/index.json (NugetFeed)

Microsoft.WindowsDesktop.App @ 5.0.0-preview.1.20127.4
Repository:    https://github.com/dotnet/windowsdesktop
Branch:        release/5.0-preview1
Commit:        80f111daf668cc7bc8c601fc34e5d559096ab043
Build Number:  20200227.4
Date Produced: 2/27/2020 12:15 PM
Build Link:    https://dev.azure.com/dnceng/internal/_build/results?buildId=539352
BAR Build Id:  44369
Released:      False
Channels:
- .NET 5 Preview 1
Locations:
- https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json (NugetFeed)
- https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5-transport/nuget/v3/index.json (NugetFeed)
```